src/modules/Laboratorio/Listar/actions/selectionStatus.js

```diff
@@ -1,12 +1,12 @@
 module.exports = function selectionStatus(status) {
```

Comment:
A sintaxe de exportação pode ser atualizada para o padrão mais atual do JavaScript, utilizando `export default`.

```suggestion
export default function selectionStatus(status) {
```